src/index.html

```diff
@@ -22,6 +22,346 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="head head--flex">
+        <div class="head__item1">
+          <a
+            class="head-logo-link"
+            href="#"
+          >
+            <img
+              src="images/logo.png"
+              alt="picture logo"
+            />
+          </a>
+        </div>
+        <div class="head__item2">
+          <nav class="head-nav">
+            <ul class="head-nav__list">
+              <li>
+                <a
+                  class="is-active"
+                  href="#"
+                >
+                  Apple
+                </a>
+              </li>
+              <li>
+                <a href="#">Samsung</a>
+              </li>
+              <li>
+                <a href="#">Smartphones</a>
+              </li>
+              <li>
+                <a
+                  href="#"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li>
+                <a href="#">Gadgets</a>
+              </li>
+              <li>
+                <a href="#">Tablets</a>
+              </li>
+              <li>
+                <a href="#">Photo</a>
+              </li>
+              <li>
+                <a href="#">Video</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <!-- блок product -->
+      <main class="container__card">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="picture product"
+          />
+          <div class="product card__prod">
+            <h3 class="product__title">
+              APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h3>
+            <p class="product__code">Product code: 195434</p>
+          </div>
+          <div class="reviews card__review">
+            <!-- блок stars -->
+            <div class="stars stars--4 reviews__stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__count">Reviews: 5</div>
+          </div>
+          <div class="price card__price">
+            <p class="price__title">Price:</p>
+            <div class="price__current">$2,199</div>
+          </div>
+          <a
+            class="btn-link btn-link--hover"
+            data-qa="card-hover"
```

Comment:
The `data-qa` attribute is used here as `card-hover`, but in other similar elements, it is just `hover`. Ensure consistency in naming conventions for data attributes to avoid confusion.